Lib/argparse.py

```diff
@@ -2613,7 +2611,7 @@ def _check_value(self, action, value):
         # converted value must be one of the choices (if specified)
         if action.choices is not None and value not in action.choices:
             args = {'value': value,
-                    'choices': ', '.join(map(repr, action.choices))}
+                    'choices': ', '.join(map(str, action.choices))}
```

Comment:
Note that `repr()` is used for `value`. If it is an enum value, you will get a weird message: `invalid choice: <Enum.FOO: 'foo'> (choose from bar, baz)`.